collect_app/src/androidTest/java/org/odk/collect/android/formentry/DeletingRepeatGroupsTest.java

```diff
@@ -0,0 +1,282 @@
+package org.odk.collect.android.formentry;
+
+import android.Manifest;
+
+import androidx.test.espresso.intent.rule.IntentsTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.odk.collect.android.R;
+import org.odk.collect.android.activities.FormEntryActivity;
+import org.odk.collect.android.espressoutils.FormEntry;
+import org.odk.collect.android.support.CopyFormRule;
+import org.odk.collect.android.support.ResetStateRule;
+import org.odk.collect.android.support.matchers.RecyclerViewMatcher;
+import org.odk.collect.android.test.FormLoadingUtils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.longClick;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+
+public class DeletingRepeatGroupsTest {
+    private static final String TEST_FORM = "repeat_groups.xml";
+
+    @Rule
+    public IntentsTestRule<FormEntryActivity> activityTestRule = FormLoadingUtils.getFormActivityTestRuleFor(TEST_FORM);
+
+    @Rule
+    public RuleChain copyFormChain = RuleChain
+            .outerRule(GrantPermissionRule.grant(
+                    Manifest.permission.READ_EXTERNAL_STORAGE,
+                    Manifest.permission.WRITE_EXTERNAL_STORAGE
+                    )
+            )
+            .around(new ResetStateRule())
+            .around(new CopyFormRule(TEST_FORM, null));
+
+    @Test
+    public void testDeletingRepeatGroupsInForm() {
+        FormEntry.clickOnString(R.string.add_another);
+        FormEntry.putText("11");
+        FormEntry.swipeToNextQuestion();
+        FormEntry.putText("12");
+        FormEntry.swipeToNextQuestion();
+        FormEntry.clickOnString(R.string.add_another);
+        FormEntry.putText("21");
+        FormEntry.swipeToNextQuestion();
+        FormEntry.putText("22");
+        FormEntry.swipeToNextQuestion();
+        FormEntry.clickOnString(R.string.add_another);
+        FormEntry.putText("31");
+        FormEntry.swipeToNextQuestion();
+        FormEntry.putText("32");
+        FormEntry.swipeToNextQuestion();
+        FormEntry.clickOnString(R.string.add_another);
+        FormEntry.putText("41");
+        FormEntry.swipeToNextQuestion();
+        FormEntry.putText("42");
+        FormEntry.swipeToPrevoiusQuestion();
```

Comment:
Would be good to get this typo fixed! Could be as a commit here or in another PR. (`Prevoius` -> `Previous`)

Reply:
Fixed.